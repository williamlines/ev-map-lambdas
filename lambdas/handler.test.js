const { hello } = require("./handler");

describe("handler", () => {
  it("should not throw", async () => {
    const ret = await hello();
    expect(ret).toMatchSnapshot();
  });
});
