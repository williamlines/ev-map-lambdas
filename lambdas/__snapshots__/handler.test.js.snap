// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`handler should not throw 1`] = `
{
  "body": "{
  "message": "Go Serverless v3.0! Your function executed successfully!"
}",
  "statusCode": 200,
}
`;
